packages/ui5-application-inquirer/test/unit/prompts/prompts.test.ts

```diff
@@ -303,7 +303,7 @@ describe('getQuestions', () => {
 
         // This choice is not a maintained version and so the closest maintained version should be added
         const defaultChoice = {
-            'name': '1.120.99',
+            'name': '1.120.99 (Source system version)',
```

Comment:
What happens if a default choice is passed with out an additional label. I think the (`Maintained`) string will not be shown anymore.

Reply:
Yes (Maintained version) is not shown as you expect 

![image](https://github.com/user-attachments/assets/ae0e8006-b370-46cf-9819-56a203621f14)

Reply:
OK to override as long as we document. Also in that case can we bunp minor version instead of patch also.